knowledge-system/scripts/train_cks_full.py

```
@@ -0,0 +1,209 @@
+    conn.commit()
+    
+    print(f"✅ Updated {updated} components, inserted {inserted} new components")
+    
+    # Show statistics
+    cursor.execute('SELECT COUNT(*) FROM components')
+    total_components = cursor.fetchone()[0]
+    
+    cursor.execute('SELECT COUNT(DISTINCT language) FROM components')
+    languages = cursor.fetchone()[0]
+    
+    print(f"\n📊 Knowledge Base Statistics:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)